plugins/vela.yaml

```diff
@@ -3,14 +3,14 @@ kind: Plugin
 metadata:
   name: vela
 spec:
-  version: "v1.0.4"
+  version: "v1.1.0-alpha.1"
```

Comment:
We generally don't recommend to release alpha versions, because krew offers no way to rollback to an earlier version. Effectively, you are releasing your software, so maybe it isn't alpha?

Reply:
we can close this PR, we'll release 1.1 soon, and this release is for someone who want to take a fresh try.

Reply:
Ok thanks for clarifying.